samples/scaleout/senderside-and-distributor/Core_6/Worker.Handlers/ProcessOrderCommandHandler.cs

```diff
@@ -0,0 +1,30 @@
+﻿using NServiceBus;
+using System.Threading.Tasks;
+using NServiceBus.Logging;
+
+public class ProcessOrderCommandHandler :
+    IHandleMessages<PlaceOrder>
+{
+    static ILog log = LogManager.GetLogger<ProcessOrderCommandHandler>();
+
+    #region SendLocal
+
+    public async Task Handle(PlaceOrder placeOrder, IMessageHandlerContext context)
+    {
+        log.Info($"Sending order {placeOrder.OrderId} to validation.");
+
+        await Task.Delay(1000).ConfigureAwait(false);
```

Comment:
To better show the users what's going on when running. Otherwise stuff shows up in the console too quick.